index.html

```diff
@@ -1177,119 +1177,269 @@ <h2 id="relationship-to-permissions-policy">
         permission=] or by some user agent policy.
       </p>
     </section>
-    <section data-cite="webdriver">
+    <section>
       <h2 id="automation">
         Automated testing
       </h2>
       <p>
         For the purposes of user-agent automation and application testing, this document defines
-        the following <a>extension commands</a> for the [[WebDriver]] specification. It is OPTIONAL
-        for a user agent to support <a>extension commands</a> commands.
+        extensions to the [[WebDriver]] and [[WebDriver-BiDi]] specifications.
+        It is OPTIONAL for a user agent to support them.
       </p>
       <pre class='idl'>
         dictionary PermissionSetParameters {
           required PermissionDescriptor descriptor;
           required PermissionState state;
         };
       </pre>
-      <section>
-        <h3 id="set-permission-command">
-          Set Permission
+      <p>To <dfn>set a permission</dfn> given <var>descriptor</var>, and <var>state</var>:
+      <ol class="algorithm">
+        <li>Let |settings| be the [=current settings object=].
+        </li>
+        <li>Let |targets| be a <a>list</a> containing all <a>environment settings objects</a>
+        whose [=environment settings object/origin=] is the <a data-lt="same origin">same</a> as
+        the [=environment settings object/origin=] of |settings|.
+        </li>
+        <li>Let |tasks| be an empty <a>list</a>.
+        </li>
+        <li>For each <a>environment settings object</a> |target| in |targets|:
```

Comment:
Where possible, please use the [= =] syntax... 

```suggestion
        <li>For each [=environment settings object=] |target| in |targets|:
```

Reply:
There are 64 occurrences of `<a>`. Better do this in another PR, agreed? I'm afraid it would be too noisy to change them to `[= =]` in this PR.

Reply:
I changed the occurrence you asked for, but won't change many others so that we don't add a lot of noise in the PR